exercises/concept/kurokos-clock/.meta/design.md

```diff
@@ -0,0 +1,39 @@
+# Design
+
+## Goal
+
+The Goal is to learn to use the `Time` module.
+We need to make it very clear that that module by itself is not enough to handle complex date and time tasks, but it merely a tool to display a time provided by the browser. More complex uses are done with packages.
+
+## Learning objectives
+
+Students should be able to
+
+- understand what POSIX time means
+- transform `Posix` into millisecond `Int` back and forth (`posixToMillis`, `millisToPosix`)
```

Comment:
Not all these things are in the exercise, but I think thats ok

Reply:
Yes, it's only mentioned in the intro.
I though that since one of the goals of the concept was to make it clear that `Time` shouldn't really be used for manipulating time, we were better off just skipping it.
I am still a bit on the fence on that, but since you are saying it's OK, let's keep it that way.